docs/antora/modules/ROOT/pages/Common_Project_Layouts.adoc

```diff
@@ -103,6 +109,11 @@ latter of which runs your code on Node.js, which must be pre-installed)
 `ScalaJSModule` also exposes the `foo.fastOpt` and `foo.fullOpt` tasks for
 generating the optimized Javascript file.
 
+ScalaJSModule requires declaring it's dependencies as third party
+libraries (note the two sets of double-colons `::` necessary between the library name and the version).
```

Comment:
Might look as nitpicking, but the wording  made me understand that double colons are needed for "thirdparty libraries" whereas they are needed for "Scala.js libraries" in general.

```suggestion
libraries (don't forget to use double-colons `::` between the library name and the version for Scala.js dependencies).
```